public/app/index.ts

```diff
@@ -22,9 +22,9 @@ import app from './app';
 
 const prepareInit = async () => {
   if (process.env.frontend_dev_mock_api) {
-    return import('test/mock-api/worker').then((workerModule) => {
-      workerModule.default.start({ onUnhandledRequest: 'bypass' });
-    });
+    // return import('test/mock-api/worker').then((workerModule) => {
```

Comment:
Any particular reason this is commented out?

Reply:
Nope, it was an accident 😁 This is how I normally turn off the mock API locally as its quicker than changing custom ini & restarting, I just accidentally committed it this time. Thanks!